src/web/routes/lifecycle-events/integration.test.ts

```diff
@@ -1,12 +1,243 @@
 import { HttpStatusCode } from 'axios';
+import nock from 'nock';
 import request from 'supertest';
+import { v4 as uuidv4 } from 'uuid';
+
+import { generateConnectLifecycleRequest } from './testing';
 
 import app from '../../../app';
+import { getConfig } from '../../../config';
+import {
+	generateConnectInstallationCreateParams,
+	generateFigmaTeam,
+	generateFigmaUserCredentialsCreateParams,
+} from '../../../domain/entities/testing';
+import {
+	connectInstallationRepository,
+	figmaTeamRepository,
+} from '../../../infrastructure/repositories';
+import { generateInboundRequestAsymmetricJwtToken } from '../../testing';
+
+const mockConnectKeyEndpoint = ({
+	baseUrl,
+	keyId,
+	publicKey,
+	status = HttpStatusCode.Ok,
+}: {
+	baseUrl: string;
+	keyId: string;
+	publicKey: string;
+	status: HttpStatusCode;
+}) => {
+	nock(baseUrl).get(`/${keyId}`).reply(status, publicKey);
+};
+
+const mockFigmaDeleteWebhookEndpoint = ({
+	baseUrl,
+	webhookId,
+	accessToken,
+	status = HttpStatusCode.Ok,
+}: {
+	baseUrl: string;
+	webhookId: string;
+	accessToken: string;
+	status: HttpStatusCode;
+}) => {
+	nock(baseUrl, {
+		reqheaders: {
+			Authorization: `Bearer ${accessToken}`,
+		},
+	})
+		.delete(`/v2/webhooks/${webhookId}`)
+		.reply(status);
+};
 
 describe('/lifecycleEvents', () => {
-	it('/installed should respond 401 when JWT token is missing', () => {
-		return request(app)
-			.post('/lifecycleEvents/installed')
-			.expect(HttpStatusCode.Unauthorized);
+	describe('/installed', () => {
```

Comment:
Thanks for adding these, think we've got integration tests on all our routes now 🎉